### PR TITLE
Document which files should be edited, and where, to remove a cluster from the federation

### PR DESCRIPTION
**Pull request by sgibson91**
This is a super quick PR to add guidance on how to remove a cluster from the federation (e.g. for maintenance). Updates, additions, rewrites are welcome!

### CODE REVIEW

**Comment by consideRatio** [comment]
LGTM but not fully confident about the trivial matter of formatting. Is it perhaps supposed to be double backtick's rather than triple in tST to accomplish what markdown does with a single backtick to make something like `like this`?

**Comment by sgibson91** [comment]
Honestly, I have no idea - I pulled up an rST cheat sheet 😅

**Comment by consideRatio** [comment]
This is how it looks at the moment based on the PR preview by RTD:

image

**Comment by sgibson91** [comment]
Then yes, it's probably two backticks - please feel free to edit directly! 😊

File: docs/source/operation_guide/federation.rst [hunk 1]
```diff
@@ -86,3 +86,23 @@ turing.mybinder.org
...
+rotation. For example, maintenence work, a problem with the deployment, and so
+on.
+
+There are 3 main files you may wish to edit in order to remove a cluster from
+the Federation:
+
+#. *Required.* Set the ```binderhub.config.BinderHub.pod_quota``` key to ```0``` in the
+   cluster's config file under the `config <https://github.com/jupyterhub/mybinder.org-deploy/tree/master/config>`_
+   directory
+#. *Recommended.* Set the ```weight``` key for the cluster to ```0``` in the
```
Comment (consideRatio) [inline docs/source/operation_guide/federation.rst:103]:
```suggestion
#. *Required.* Set the ``binderhub.config.BinderHub.pod_quota`` key to ``0`` in the
   cluster's config file under the `config <https://github.com/jupyterhub/mybinder.org-deploy/tree/master/config>`_
   directory
#. *Recommended.* Set the ``weight`` key for the cluster to ``0`` in the
```

**Review by consideRatio — APPROVED** [review]

**Pull request merged**